### DISCUSSION TITLE
Mixer volume reset to 100 in MPD settings

### DISCUSSION THREAD

**Discussion by lochbika** [rern/rAudio#85] [Fixed Issues]
Hi,
first of all, great work!
I found one detail that is a bit annoying: the mixer volume in the MPD settings page is always reset to 100 after a reboot. Before this feature was available I set it in alsamixer. However, if I do it this way the mixer volume is also reset to 100. I use the latest update 20210105.

![Screenshot_20210106_112914](https://user-images.githubusercontent.com/19695207/103758446-81aa7f00-5012-11eb-8a80-5c56c0685780.png)
![Screenshot_20210106_112941](https://user-images.githubusercontent.com/19695207/103758451-84a56f80-5012-11eb-848d-0399608895ad.png)

**Comment by rern** [comment]
Volume Control "MPD software" always sets Mixer device to 100 to optimize quality in software volume mode.
Set to Mixer device and get a better sound quality.

**Reply by lochbika** [reply]
That sounds plausible. However, I am using a amplifier board module with fixed volume (set by input volume). I find it quite nice to use the alsamixer volume as a maximum setting and use the mpd software volume as a fine adjustment. Otherwise my speakers are unbearable loud already at a mpd volume of 20 (with mixer volume at 100), and the volume jumps at low volumes are huge. 

Also: Why the speaker volume button (Mixer Device Volume) then anyway?

Cheers

**Comment by rern** [comment]
Fixed.
